services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -229,24 +230,25 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
                 retrievedRecords = items.Count();
                 this.Logger.LogDebug("Ingest [{name}] retrieved [{countOfRecordsRetrieved}] records", manager.Ingest.Name, retrievedRecords);
 
+                DateTime? lastReceivedContentOn = null;
                 if (retrievedRecords == 0 && importDateEnd.HasValue)
                 {
                     this.Logger.LogDebug("Ingest [{name}] - no records prior to import end date filter [{importDateEnd}] records", manager.Ingest.Name, importDateEnd.Value.ToString("yyyy-MM-dd h:mm:ss tt"));
-                    creationDateOfLastImport = importDateEnd.Value;
+                    lastReceivedContentOn = importDateEnd.Value;
```

Comment:
Fixed bug which would result in missing content from TNO due to the skip and date filter implementation.